test-data/run.test

```diff
@@ -636,6 +636,15 @@ f(5)
 [out]
 5
 
+[case testImportMissing]
+# pytest doesn't have stubs but it is obviously installed...
```

Comment:
Hm, test dependency on stuff we don't control...

Reply:
Maybe the right thing to do is to add support to the mypyc test harness for files that aren't typechecked.